### PR TITLE
DOC-10391: Changes 'database' to 'keyspace' and corrects grammar

### PR DESCRIPTION
**Pull request by franrob-projects**
Changes 'database' to 'keyspace' and corrects grammar

### CODE REVIEW

File: modules/ROOT/pages/changes-feed.adoc [hunk 1]
```diff
@@ -31,10 +31,9 @@ include::partial$server-integration-scenario-table.adoc[]
 == Changes Feed
 
 
-The changes-feed for the reliever returns a sorted list of changes made to documents in the database.
+The changes-feed for the reliever returns a sorted list of changes made to documents in the keyspace.
 
-This article describes how to use the changes feed API to integrate {sgw} with other backend processes.
-For instance if you have a channel called "needs-email" you could have a bot that sends an email and then saves the document back with a flag to keep it out of the "needs-email" channel.
+This article describes using the changes feed API to integrate sync gateway with other backend processes. For instance, if you have a channel called 'needs-email', you could have a bot that sends an email and then saves the document with a flag to keep it out of the 'needs-email' channel.
```
Comment (dinaabennett) [inline modules/ROOT/pages/changes-feed.adoc]: Does this need to be on two lines?

File: modules/ROOT/pages/changes-feed.adoc [hunk 2]
```diff
@@ -31,10 +31,9 @@ include::partial$server-integration-scenario-table.adoc[]
 == Changes Feed
 
 
-The changes-feed for the reliever returns a sorted list of changes made to documents in the database.
+The changes-feed for the reliever returns a sorted list of changes made to documents in the keyspace.
```
Comment (dinaabennett) [inline modules/ROOT/pages/changes-feed.adoc]: reliever? is that a term, or do we mean receiver or something else?

**Review by dinaabennett — APPROVED** [review]
added comments :-)

File: modules/ROOT/pages/changes-feed.adoc (hunk 2 above)
Comment (franrob-projects) [reply modules/ROOT/pages/changes-feed.adoc]: Let's ask GPT as s first shot

File: modules/ROOT/pages/changes-feed.adoc [hunk 3]
```diff
@@ -31,10 +31,10 @@ include::partial$server-integration-scenario-table.adoc[]
 == Changes Feed
 
 
-The changes-feed for the reliever returns a sorted list of changes made to documents in the database.
+The changes-feed for the receiver returns a sorted list of changes made to documents in the keyspace.
```
Comment (osfameron) [inline modules/ROOT/pages/changes-feed.adoc:34]: I'm relieved.

**Review by osfameron — APPROVED** [review]
👍

File: modules/ROOT/pages/changes-feed.adoc (hunk 3 above)
Comment (dinaabennett) [reply modules/ROOT/pages/changes-feed.adoc:34]: well received!

Comment (franrob-projects) [reply modules/ROOT/pages/changes-feed.adoc:34]: RIP'd

**Pull request merged**